src/components/actions/evaluatorFunctions.ts

```diff
@@ -46,10 +46,21 @@ const getISODate = (date?: Date) =>
   date ? DateTime.fromJSDate(date).toISO() : DateTime.now().toISO()
 
 // Extracts any numeric content from a string
-export const extractNumber = (input: string) => {
+const extractNumber = (input: string) => {
   const numberMatch = input.match(/(-?(\d+\.\d+))|(-?((?<!\.)\.\d+))|(-?\d+)/gm)
   if (!numberMatch) return 0
   return Number(numberMatch[0])
 }
 
-export default { generateExpiry, getYear, getFormattedDate, getJSDate, getISODate, extractNumber }
+// Remove diacritics (accented characters) from strings
+const removeAccents = (input: string) => input.normalize('NFD').replace(/[\u0300-\u036f]/g, '')
```

Comment:
Not critical, but we'll need this if we want to implement https://github.com/openmsupply/conforma-web-app/issues/1388

(Will still need config in filter_definitions, but this will be needed too)